addons/l10n_ar/models/res_company.py

```diff
@@ -16,7 +16,7 @@ class ResCompany(models.Model):
     l10n_ar_afip_responsibility_type_id = fields.Many2one(
         domain="[('code', 'in', [1, 4, 6])]", related='partner_id.l10n_ar_afip_responsibility_type_id', readonly=False)
     l10n_ar_company_requires_vat = fields.Boolean(compute='_compute_l10n_ar_company_requires_vat', string='Company Requires Vat?')
-    l10n_ar_afip_start_date = fields.Date('Activities Start Date')
+    l10n_ar_afip_start_date = fields.Date('Activities start')
```

Comment:
@jco-odoo when we were working on the report we found out this  field name was an error. Because what we want to express there is "Inicio de Actividades" Activities Start, So we add it to the report and we change it in the model just to make them equal for the user. I have updated the label an now is capitalized. @jjscarafia please mark this comment as resolved